api/.env

```diff
@@ -74,7 +74,7 @@ OAUTH_JUBLADB_BASE_URL="https://jubla.puzzle.ch"
 
 ###> sentry/sentry-symfony ###
 SENTRY_API_DSN=""
-SENTRY_ENVIRONMENT="http://localhost:3000"
+SENTRY_ENVIRONMENT="local"
```

Comment:
was this change on purpose?

Reply:
Yes. Sentry doesn't like full URLs as environment